roles/keystone/tasks/monitoring.yml

```diff
@@ -17,6 +17,19 @@
                args="--service keystone --criticality {{ keystone.monitoring.sensu_checks.check_keystone_api.criticality }}"
   notify: restart sensu-client
 
+- block:
+  - name: create directory for below symbolic link
+    file:
+      path: /opt/openstack/current/keystone/bin/
+      state: directory
+  
+  - name: add symolic link for keystone expired token check (rhel distro only)
+    file: 
+      src: /bin/python
+      dest: /opt/openstack/current/keystone/bin/python
```

Comment:
Why do we need this ?  is there a python path hard coded in the check ?  of so we should fix that, not create a weird workaround like this.